journal/updated/01-journal.adoc

```diff
@@ -767,251 +767,270 @@ nevertheless, afterward it yields the peaceable fruit of righteousness,
+nor made any acquaintance with her, nor did I know when or where I would see her again.
+I was very willing to let the Lord order it as it seemed best to Himself,
+and therein I had peace of mind; and in time the Lord brought
+us into acquaintance with one another,
 and she confessed that she had some sight of the
 same thing that I had seen concerning her.
 So after some time we parted, and I was freely resigned to the will of God;
-and when we came together again, I told her, if the Lord did order her to be my wife,
+and when we came together again, I told her that if the Lord ordered her to be my wife,
 she must come with me to a strange country,
-where there were no Friends but what God in time might
-call and gather to himself. Upon a little consideration,
-she said, if the Lord should order it so, she must go with her husband,
-though it were to the wilderness;
+where there were no Friends, beside what God in time might
+call and gather to Himself. Upon a little consideration,
+she said that if the Lord ordered it so, she must go with her husband,
+even if it were into the wilderness;
 and being somewhat sensible of the workings of God upon her spirit in this matter,
-she was willing to condescend in her mind to what he wrought in her;
-but by hearkening to one who had not well weighed the matter,
-she became disobedient to what God had revealed to her;
+she was willing to condescend in her mind to what He wrought in her.
+But by hearkening to one who had not well weighed the matter,
+she became disobedient to what God had revealed to her,
 which brought great sorrow and trouble upon her.
 I went to see her in this poor condition,
 and I rested satisfied with the will of God in this concern, being freely resigned,
-if the Lord had wrought the same thing in her, as was in me,
-to receive her as his gift to me; and after some time, we waiting upon the Lord together,
-she arose, and declared before me,
-and the other Friend who had begot doubts and reasonings in her mind,
-That in the name and power of God she consented to be my wife, and to go along with me,
-wherever the Lord should order us; and I said, in the fear of the Lord,
-I receive you as the gift of God to me.
-So I rested satisfied in the will of God, for a further accomplishment of it.
-
-Under a weighty consideration, which way to take each other in marriage,
+if the Lord had wrought the same thing in her as was in me,
+to receive her as His gift to me. And after some time,
+we waiting upon the Lord together, she arose and declared before me,
+and the other Friend who had planted doubts and reasonings in her mind,
+that in the name and power of God she consented to be my wife, and to go along with me
+wherever the Lord ordered us; and I said, in the fear of the Lord,
+"`I receive you as the gift of God to me.`"
+So I rested satisfied in the will of God for a further accomplishment of it.
+
+Under the weighty consideration of the manner in which to take each other in marriage,
```

Comment:
maybe just add: of the manner in which **we should** take each other in marriage